frontend/src/data/variables/AcsPopulationProvider.ts

```diff
@@ -35,17 +35,49 @@ class AcsPopulationProvider extends VariableProvider {
     super(
       "acs_pop_provider",
       ["population", "population_pct"],
-      ["acs_population-by_race_state_std", "acs_population-by_age_state"]
+      [
+        "acs_population-by_race_state_std",
+        "acs_population-by_race_county_std",
+        "acs_population-by_age_state",
+      ]
     );
   }
 
+  getDatasetId(breakdowns: Breakdowns): string {
+    if (breakdowns.demographicBreakdowns.age.enabled) {
+      return "acs_population-by_age_state";
+    }
+    if (
+      breakdowns.demographicBreakdowns.race_nonstandard.enabled ||
+      breakdowns.demographicBreakdowns.race.enabled
+    ) {
+      return breakdowns.geography === "county"
+        ? "acs_population-by_race_county_std"
+        : "acs_population-by_race_state_std";
+    }
+    return "";
```

Comment:
I'd probably throw an error here - otherwise it'll just throw an error anyway when you try to use an invalid id, but the error will be clearer here